charts/minikube-values.yaml

```diff
@@ -2,13 +2,17 @@
 # Configuration file for deploying the renga chart on minikube
 #
 
+global:
+  renga:
+    domain: renga-k8s.build
+
 ingress:
   enabled: true
   annotations:
     kubernetes.io/ingress.class: nginx
     nginx.ingress.kubernetes.io/ssl-redirect: false
   hosts:
-  - ""
+  - "renga-k8s.build"
```

Comment:
missing the new host?

Reply:
@leafty I don't understand the question. Can you please clarify it?

Reply:
i'm expecting the ingress to reject HTTP requests for host `gitlab.renga-k8s.build` (!= `renga-k8s.build`)

Reply:
Even with line https://github.com/SwissDataScienceCenter/renga/pull/190/files#diff-36d359d0ba9a7120256f9209a50eb5faR50 ?

Reply:
ok. but then the value is also used for the TLS config, with auto LE with `cert-manager`